packages/core/src/mesh/PrimitiveMesh.ts

```diff
@@ -20,21 +15,24 @@ export class PrimitiveMesh {
    * @param engine - Engine
    * @param radius - Sphere radius
    * @param segments - Number of segments
+   * @param accessible - Whether to access data later. If true, you can access the data anytime
    * @returns Sphere mesh
    */
-  static createSphere(engine: Engine, radius: number = 0.5, segments: number = 12): Mesh {
-    const mesh = new Mesh(engine);
+  static createSphere(
+    engine: Engine,
+    radius: number = 0.5,
+    segments: number = 12,
+    accessible: boolean = false
+  ): ModelMesh {
+    const mesh = new ModelMesh(engine, "sphereModelMesh");
```

Comment:
"sphereModelMesh":First letter should be capitalized and you can simplfy to "Sphere". For example you are a human, but your name is just "ChengKong",not "ChengKongHuman"